packages/service/fixtures/hello-client-from-plugin/plugin.js

```diff
@@ -0,0 +1,9 @@
+'use default'
+
+/**  @type {import('fastify').FastifyPluginAsync<{ optionA: boolean, optionB: string }>} */
+module.exports = async function (app) {
+  const result = app.hello.get()
```

Comment:
So this _should_ work, but I don't quite understand why it won't

Reply:
You'd need to call `app.configureHello()`, as the client is not decorated to the app by default: https://github.com/platformatic/platformatic/blob/c5c9db29852a4c9d87fa86801480d23f1d65da06/packages/client/index.js#L426.

I think we removed this for some reason, but I can't remember why.
Maybe @leorossi or @rozzilla remember.

Reply:
I updated the test to just check if `app.configureHello` is defined.